### PR TITLE
moving directories around and adding web directory

### PR DESCRIPTION
**Pull request by hfekete**
moving directories around and adding web directory

### CODE REVIEW

File: btr-api/LICENSE
Comment (thorwolpert) [inline btr-api/LICENSE:1]: We can probably leave the license at the root level?

File: btr-web/.npmrc
Comment (thorwolpert) [inline btr-web/.npmrc:1]: this should be at the root?

File: btr-web/common-layer/.npmrc
Comment (thorwolpert) [inline btr-web/common-layer/.npmrc:1]: only 1 at the root level is needed

Comment (thorwolpert) [reply btr-web/common-layer/.npmrc:1]: ah, i see you've moved all the pnpm stuff under web - nice!

File: btr-web/common-layer/package.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    "@nuxtjs/tailwindcss": "^6.8.0",
+    "@pinia/nuxt": "^0.4.11",
+    "cypress": "^13.1.0",
+    "eslint": "^8.48.0",
+    "nuxt": "^3.7.0",
+    "nuxt-vitest": "^0.10.4",
+    "typescript": "^5.2.2",
+    "vitest": "^0.33.0"
+  },
+  "simple-git-hooks": {
```
Comment (thorwolpert) [inline btr-web/common-layer/package.json:37]: nice!

File: migrations/__pycache__/env.cpython-311.pyc
Comment (thorwolpert) [inline migrations/__pycache__/env.cpython-311.pyc:1]: __pycache__ should be ignored

File: btr-web/common-layer/package.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{
+  "name": "nuxt-app",
```
Comment (thorwolpert) [inline btr-web/common-layer/package.json:2]: pnpm can filter on names, so I like to name this after the layer being used, maybe `common-layer`

**Review by thorwolpert — COMMENTED** [review]
A few comments.
Also, add in the main web app and extend the nuxt layer. 
I do that to ensure I have my pnpm stuff installed and working as expected across multiple packages

File: btr-api/.dockerignore [hunk 3]
```diff
@@ -1,6 +1,6 @@
 Dockerfile
 .dockerignore
-.env*
```
Comment (kialj876) [inline btr-api/.dockerignore:3]: we want to ignore .env too

**Comment by kialj876** [comment]
Are you going to add in the header / footer / common css (probably pull this from the namex project) to the common layer? Also I was thinking this would include a landing page for BTR that would be empty for now, but using the common layer stuff so that we can see that working. And we should have the github CI running for the lint / tests so that we know that part is good to go too right?

File: btr-api/Dockerfile [hunk 4]
```diff
@@ -83,7 +83,7 @@ USER web
 
 # The following stage is only for production:
 FROM development_build AS production_build
-COPY --chown=web:web . /code
+COPY --chown=web:web .. /code
```
Comment (hfekete) [inline btr-api/Dockerfile:86]: Can someone verify this is correct, or it got messed up during moving process.

File: btr-api/.devcontainer/docker-compose.yml [hunk 5]
```diff
@@ -3,8 +3,8 @@ version: '3.8'
 services:
   app:
     build:
-      context: ..
-      dockerfile: .devcontainer/Dockerfile
+      context: ../..
```
Comment (hfekete) [inline btr-api/.devcontainer/docker-compose.yml:6]: Also, not sure if this got messed up, or it should be like that.

File: btr-api/Dockerfile (hunk 4 above)
Comment (kialj876) [reply btr-api/Dockerfile:86]: Hmm I think this shouldn't have changed -- did you try building it?

File: btr-api/.devcontainer/docker-compose.yml (hunk 5 above)
Comment (kialj876) [reply btr-api/.devcontainer/docker-compose.yml:6]: same with this file -- should probably be the same paths as before I think? Try running it and see what happens

**Pull request closed without merging**